src/astro/utils/snowflake_transform.py

```diff
@@ -10,8 +10,8 @@ def _handle_table(t: Table):
     :return:
     """
 
-    snow_schema = t.schema or SCHEMA
-    return t.database + "." + snow_schema + "." + t.table_name
+    snow_schema = getattr(t.metadata, "schema", None) or SCHEMA
+    return f"{getattr(t.metadata, 'database', None)}.{snow_schema}.{t.name}"
```

Comment:
@utkarsharma2 I think this would return `.schema.name` if there is no database. Please make sure this is tested.

Reply:
Yup this will become `None.schema.name` and raise a "AttributeError: 'NoneType' object has no attribute 'schema'" error

Reply:
This is handled now. Thanks for pointing out.